edx/edx-app-ios
# Remove MockNav from My Videos

### Description

[MA-2807](https://openedx.atlassian.net/browse/MA-2807)

Mock navigation has been removed from My Videos, and native navigation bar is not used instead. All navigation elements have also been assigned accessibility labels as needed.

### Notes

N/A

### How to test this PR

- Turn on VO
- Navigate to My Videos
- Items in navigation bar should be pronounced in consistency with other screens.

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
